arrow-fx-rx2/src/test/kotlin/arrow/fx/FlowableKTests.kt

```diff
@@ -153,12 +143,12 @@ class FlowableKTests : RxJavaSpec() {
 
     "FlowableK cancellable should cancel CancelToken on dispose" {
       Promise.uncancellable<ForFlowableK, Unit>(FlowableK.async()).flatMap { latch ->
-          FlowableK {
-            FlowableK.cancellable<Unit>(fa = {
-              latch.complete(Unit)
-            }).flowable.subscribe().dispose()
-          }.flatMap { latch.get() }
-        }.value()
+        FlowableK {
```

Comment:
Was this not properly checked by `ktlintCheck`? 🤔

Reply:
Seems not, no idea why tho 😕